scripts-dev/create_postgres_db

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
```

Comment:
this is an extraordinarily complicated way to create a postgres database.

I think what you mean is the shell one-liner:

```
PGPASSWORD=postgres createdb -h postgres -u postgres synapse
```